polaris-react/src/components/ContentBlock/tests/ContentBlock.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {mountWithApp} from 'tests/utilities';
+
+import {ContentBlock} from '../ContentBlock';
+
+const text = 'This is a stack';
+const children = <p>{text}</p>;
+
+describe('<ContentBlock />', () => {
+  it('renders children', () => {
+    const contentBlock = mountWithApp(
+      <ContentBlock width="medium">{children}</ContentBlock>,
+    );
+
+    expect(contentBlock).toContainReactComponent('p', {children: text});
+  });
+
+  it('renders custom properties', () => {
+    const contentBlock = mountWithApp(
+      <ContentBlock width="large">{children}</ContentBlock>,
```

Comment:
If we update `Width` type in `ContentBlock`, this needs to be updated
```suggestion
      <ContentBlock width="lg">{children}</ContentBlock>,
```